README.md

```diff
@@ -279,6 +259,7 @@ I would like to thank...
   - [Answer_2]
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
+-->
 
 <!-- LICENSE -->
```

Comment:
- [ ] Great job making your README descriptive as requested.  Unfortunately, when I click on the MIT license link, it gives me an error 404 page. It is not professional to allow broken links in our codebase. Kindly fix this issue.
![image](https://github.com/OnlyUnknown/vet-clinic/assets/62849872/406d1819-aa62-4f37-b8a5-0fe134ba7967)